sql/core/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -84,7 +84,7 @@ class TypedColumn[-T, U](
  *   col("`a.column.with.dots`") // Escape `.` in column names.
  *   $"columnName"               // Scala short hand for a named column.
  *   expr("a + 1")               // A column that is constructed from a parsed SQL Expression.
- *   lit("1")                    // A column that produces a literal (constant) value.
+ *   lit("abc")                  // A column that produces a literal (constant) value.
```

Comment:
cc @marmbrus  I changed it to a string literal. Otherwise it might be confusing to some users thinking the way to specify a numeric literal is also to wrap it in string.